components/src/preact/mutationFilter/mutation-filter.tsx

```diff
@@ -102,30 +101,33 @@ export const MutationFilterInner: FunctionComponent<MutationFilterInnerProps> =
     };
 
     return (
-        <div class={`h-full w-full rounded-lg border border-gray-300 bg-white p-2 overflow-scroll`}>
-            <div class='flex justify-between'>
+        <form className='w-full border boder-gray-300 rounded-md relative' onSubmit={handleSubmit} ref={formRef}>
+            <div className='absolute -top-3 -right-3'>
+                <Info height={'100px'}>Info for mutation filter</Info>
+            </div>
+            <div className='w-full flex p-2 flex-wrap items-center'>
                 <SelectedMutationDisplay
                     selectedFilters={selectedFilters}
                     setSelectedFilters={setSelectedFilters}
                     fireChangeEvent={fireChangeEvent}
                 />
-                <Info>Info for mutation filter</Info>
-            </div>
-
-            <form className='mt-2 w-full' onSubmit={handleSubmit} ref={formRef}>
-                <label className={`input flex items-center gap-2 ${isError ? 'input-error' : 'input-bordered'}`}>
+                <div
+                    className={`w-full flex focus:outline-none focus:ring-0 border ${isError ? 'border-red-500' : 'border-gray-300'} border-solid m-2 text-sm`}
+                >
                     <input
-                        className='grow min-w-0'
+                        className='grow'
```

Comment:
```suggestion
                        className='grow input'
```

or something similar, so that the text in the field gets a little marging to the border. (The `+` button needs to be centered then again)